examples/step-86/doc/results.dox

```diff
@@ -0,0 +1,243 @@
+<h1>Results</h1>
+
+When you run this program with the input file as is, you get output as follows:
+@code
+===========================================
+Number of active cells: 768
+Number of degrees of freedom: 833
+
+Time step 0 at t=0
+     5 linear iterations.
+     8 linear iterations.
```

Comment:
I presume the first run is with `beuler` with no time step adaption. 
In this case there should be only one linear solve per step.
So, why do you get two linear solves? Is this already adapting the mesh? 
If so, it should probably be discussed

Reply:
Yes, this is with `beuler` as chosen in the input file. Time step 2 also requires 2 solves. I have to admit that I don't have any idea why this is -- help appreciated! (My best guess: The solver has no idea that the equation is linear, and so runs a nonlinear iteration; it should get a zero residual in that case, though.)

Reply:
ah, good  point. Yes, it could very well be. Maybe you can add `-snes_monitor` and rerun, explaining this.

Reply:
@stefanozampini I think I need help from a PETSc expert :-) `-snes-monitor` has no effect here, I get no additional output :-(